fleetshard/pkg/central/reconciler/reconciler.go

```diff
@@ -685,6 +814,22 @@ func (r *CentralReconciler) collectReconciliationStatus(ctx context.Context, rem
 	return status, nil
 }
 
+func (r *CentralReconciler) ensureCentralDeclarativeConfigSecretCleaned(ctx context.Context, namespace string) error {
+	declarativeConfigSecret := &corev1.Secret{}
+
+	if err := r.client.Get(ctx, ctrlClient.ObjectKey{Namespace: namespace, Name: sensibleDeclarativeConfigSecretName},
+		declarativeConfigSecret); err != nil {
+		if apiErrors.IsNotFound(err) {
+			return nil
+		}
+		return fmt.Errorf("failed to retrieve declarative config secret %s/%s: %w",
+			sensibleDeclarativeConfigSecretName, namespace, err)
```

Comment:
```suggestion
			namespace, sensibleDeclarativeConfigSecretName, err)
```
Worth checking in other places